docs/config.md

```diff
@@ -1843,7 +1843,7 @@ This value is also used to scale the limit on various resources at various scope
 when the default limits (discussed in [libp2p resource management](./libp2p-resource-management.md)) are used.
 For example, increasing this value will increase the default limit for incoming connections.
 
-Default: `[TOTAL_SYSTEM_MEMORY]/4`
+Default: `[TOTAL_SYSTEM_MEMORY]/2`
```

Comment:
:thought_balloon: 

:+1:  I know @2color run into issues when he was running on a box that had 2 GB RAM on https://fly.io/ box, and his libp2p stack only got 1/4 (512MB).  

This should improve default behavior in such setups (allocating 1GB).